crates/store/src/db/tests.rs

```diff
@@ -187,6 +187,7 @@ fn sql_select_notes() {
             .unwrap(),
             details: Some(vec![1, 2, 3]),
             merkle_path: MerklePath::new(vec![]),
+            nullifier: None,
```

Comment:
How do I create a dummy `Nullifier`?

I think our database tests could use some/a-lot of improving, half the problem seems to be that's its a pain to generate reasonable data to input/expect on.

Reply:
Would `Digest::default().into()` not work? We could also add `default()` or `dummy()` to `Nullifier` in `miden-base`.

Reply:
I found

```rust
fn num_to_nullifier(n: u64) -> Nullifier {
    Nullifier::from(num_to_rpo_digest(n))
}
```

which will work for now. The issue with these free standing functions (and `into`) is that they're impossible to find naturally. Ideally one types `Nullifier::` and let auto-complete show you the options on what's possible.

Reply:
Yeah - ideally this would be something like:
```rust
impl Nullifier {

    #[cfg(any(feature = "testing", test))]
    pub fn dummy(n: u64) -> Self {
        Self::from(num_to_rpo_digest(n))
   }
}
```
Want to make a small PR in `miden-base`?